app/src/main/java/de/tum/in/tumcampusapp/component/other/settings/SettingsFragment.kt

```diff
@@ -40,7 +40,6 @@ import io.reactivex.Single
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.CompositeDisposable
 import io.reactivex.schedulers.Schedulers
-import kotlinx.android.synthetic.main.activity_setup_eduroam.*
```

Comment:
This doesn't look correct? Do we no longer have the setup eduroam setting?

Reply:
I also felt like this was suspicious, but I checked and this was marked as unused by the IDE even in 29ef2c56b47383ef48101e72b666a8d0c7a67ebe (Release 3.12), which was pushed half a year ago.

Reply:
Okay so it seems like this import was introduced in c60552d6cff11d0f1d6db0640c3dd6d8a1e3d2ec about a year ago. Seems like it is just a residue from someone playing around with a view and removing that before committing, but forgetting to remove the import aswell.

Reply:
Ack, thanks for fishing it out!